Homeworks/Homework_2/Task_2/ILazy.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Task_2
+{
+    /// <summary>
+    /// Lazy interface.
+    /// </summary>
+    /// <typeparam name="T">Type of the supplier's return value.</typeparam>
+    public interface ILazy<T>
```

Comment:
```suggestion
    public interface ILazy<out T>
```
Потому что почему нет :)